recipes/python-node-semver/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "node-semver" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "c32bfc976fd9e003c4a15665e5fe9f337366ba6b60aeb34e4479da9d7bbb0081" %}
+
+package:
+  name: 'python-{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```

Comment:
`setuptools` isn't actually needed at runtime, from what I can tell. I'll drop it for you and merge.